### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by Amandeep-sinha**
Added :  3 X 3 grids

check live at  :-
https://hardcore-aryabhata-201d65.netlify.app/

### CODE REVIEW

File: List.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
+@import "/ComponentLibrary/Css/main.css";
+
+:root{
+    /* Colors  */
+     --primary-color:#420039;
+     --secondary-color:#E1DEE9 ;
+     --plain-primary:white;
+     --danger:red;
```
Comment (kiranpratyush) [inline List.css]: we can write --color-danger here

File: Maodal.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        <footer>
+            <div class="footer-div">
+                <a href="https://github.com/Amandeep-sinha " class="social-icons" target="_blank">  <img src="https://cdn3.iconfinder.com/data/icons/inficons/512/github.png" alt="icon" ></a>
+                <a href="https://www.linkedin.com/in/amandeep-sinha-1257251a2" target="_blank" class="social-icons"> <img src="https://cdn-icons-png.flaticon.com/512/61/61109.png" alt="icon" ></a>
+                <a href="https://amandeepcl.netlify.app" class="social-icons" target="_blank"> <img src="https://cdn1.vectorstock.com/i/1000x1000/22/45/portfolio-icon-vector-2182245.jpg" alt="icon" ></a>
+            </div>
+            Developed and Designed by : Amandeep<br><br>
+        </footer>
+</body>
+<style>
```
Comment (kiranpratyush) [inline Maodal.html]: we can move style to css file

File: Navigation.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        <footer>
+            <div class="footer-div">
+                <a href="https://github.com/Amandeep-sinha " class="social-icons" target="_blank">  <img src="https://cdn3.iconfinder.com/data/icons/inficons/512/github.png" alt="icon" ></a>
+                <a href="https://www.linkedin.com/in/amandeep-sinha-1257251a2" target="_blank" class="social-icons"> <img src="https://cdn-icons-png.flaticon.com/512/61/61109.png" alt="icon" ></a>
+                <a href="https://amandeepcl.netlify.app" class="social-icons" target="_blank"> <img src="https://cdn1.vectorstock.com/i/1000x1000/22/45/portfolio-icon-vector-2182245.jpg" alt="icon" ></a>
+            </div>
+            Developed and Designed by : Amandeep<br><br>
+        </footer>
+</body>
+<style>
```
Comment (kiranpratyush) [inline Navigation.html]: we can move style to external CSS

File: Rating.css [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+     --danger:red;
+     --success:green;
+ 
+    /* Typography */
+    --designed-font: 'Kaushan Script', cursive;
+ 
+ }
+
+ .select{
+     color: rgb(236, 164, 8);
```
Comment (kiranpratyush) [inline Rating.css]: we can give variable names

**Review by kiranpratyush — APPROVED** [review]
I am approving.But please consider the required changes

File: Grid.html [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Grid</title>
+    <link href="https://fonts.googleapis.com/css2?family=Nunito&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="/ComponentLibrary/Css/main.css">
+
```
Comment (sakshamAK) [inline Grid.html:11]: I guess you have not linked your css file with grids
image

File: Navigation.html (hunk 3 above)
Comment (Amandeep-sinha) [reply Navigation.html]: Will work on that thanks for reviewing

File: Grid.html (hunk 5 above)
Comment (Amandeep-sinha) [reply Grid.html:11]: Yeah saksham will resolve that thanks for reviewing and pointing out.

**Pull request merged**